recipes/circle-fit/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "circle-fit" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: d4173696cca074c9e5a2791169854064730b0fa53836326c55634d947222d794
+
+build:
+  number: 0
```

Comment:
It looks like this could be `noarch: python`:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python